cargo-near/src/commands/build/mod.rs

```diff
@@ -0,0 +1,50 @@
+pub mod context {
+    #[derive(Debug, Clone)]
+    pub struct Context;
+
+    impl From<Context> for cargo_near_build::BuildContext {
+        fn from(_value: Context) -> Self {
+            Self::Build
+        }
+    }
+
+    impl Context {
+        pub fn from_previous_context(
+            _previous_context: near_cli_rs::GlobalContext,
+            _scope: &<super::actions::Actions as interactive_clap::ToInteractiveClapContextScope>::InteractiveClapContextScope,
+        ) -> color_eyre::eyre::Result<Self> {
+            Ok(Self)
+        }
+    }
+}
+
+pub mod actions {
+    pub mod non_reproducible_wasm;
+    pub mod reproducible_wasm;
+
+    use strum::{EnumDiscriminants, EnumIter, EnumMessage};
+
+    #[derive(Debug, Clone, EnumDiscriminants, interactive_clap::InteractiveClap)]
+    #[strum_discriminants(derive(EnumMessage, EnumIter))]
+    #[interactive_clap(input_context = near_cli_rs::GlobalContext)]
+    #[interactive_clap(output_context = super::context::Context)]
+    pub enum Actions {
+        #[strum_discriminants(strum(
+            message = "non-reproducible-wasm  - Runs on current filesystem state without many restrictions"
+        ))]
+        /// Runs on current filesystem state without many restrictions
+        NonReproducibleWasm(self::non_reproducible_wasm::BuildOpts),
+        #[strum_discriminants(strum(
+            message = "reproducible-wasm      - Requires `docker` config added and (git)committed to Cargo.toml, runs on clean (git)working tree state"
+        ))]
+        /// Requires `docker` config added and (git)committed to Cargo.toml, runs on clean (git)working tree state
```

Comment:
I think we need to be more direct for the users to be able to make the right decision (for dev vs for prod):

```suggestion
        #[strum_discriminants(strum(
            message = "non-reproducible-wasm  - Fast and simple (recommended for use during local development)"
        ))]
        /// Fast and simple (recommended for use during local development)
        NonReproducibleWasm(self::non_reproducible_wasm::BuildOpts),
        #[strum_discriminants(strum(
            message = "reproducible-wasm      - Requires [reproducible_build] section in Cargo.toml, and all changes committed and pushed to git (recommended for the production release)"
        ))]
        /// Requires [reproducible_build] section in Cargo.toml, and all changes committed and pushed to git (recommended for the production release)
```